dotnet/WPF/VisualLayerIntegration/VisualLayerIntegration/BarGraphHostControl.xaml.cs

```diff
@@ -70,44 +69,44 @@ private void BarGraphHostControl_Loaded(object sender, RoutedEventArgs e)
             // If the user changes the DPI scale setting for the screen the app is on,
             // the CompositionHostControl is reloaded. Don't redo this set up if it's
             // already been done.
-            if (compositionHost is null)
+            if (_compositionHost == null)
```

Comment:
This check and the comment is now redundant. In the constructor you know that _compositionHost is null.

Reply:
updated